src/Core/Utility.re

```diff
@@ -683,3 +701,15 @@ module ChunkyQueue: {
   let toList = ({front, rear, _}) =>
     front @ (Queue.toList(rear) |> List.concat);
 };
+
+module Path = {
+  // Not very robust path-handling utilities.
+  // TODO: Make good
+
+  let toRelative = (~base, path) => {
+    let base = base == "/" ? base : base ++ Filename.dir_sep;
+    Str.replace_first(Str.regexp_string(base), "", path);
+  };
+
+  let explode = String.split_on_char(Filename.dir_sep.[0]);
+};
```

Comment:
We could consider using the `Fp`/`Path` library from reason-native: https://github.com/facebookexperimental/reason-native/tree/master/src/fp
(if it makes sense)

Reply:
Yeah, that looks pretty good. It doesn't seem to have any way of destructuring it like `explode` though. But we could contribute that.